ext/native.go

```diff
@@ -273,12 +275,12 @@ func (tp *nativeTypeProvider) FindStructFieldType(typeName, fieldName string) (*
 		Type: celType,
 		IsSet: func(obj any) bool {
 			refVal := reflect.Indirect(reflect.ValueOf(obj))
-			refField := valueFieldByName(tp.options.parseStructTags, refVal, fieldName)
```

Comment:
`valueFieldByName` is (I think) not needed, since we've already found the structTag to field mapping above via:
`refField, isDefined := t.hasField(fieldName)` we can just use `refField.Name` to do a direct lookup without needing to re-parse tags.

Reply:
That makes sense. Thanks.